addons/crm/models/crm_lead.py

```diff
@@ -1984,6 +1983,7 @@ def _pls_get_lead_pls_values(self, domain=[]):
         """
         leads_values_dict = OrderedDict()
         pls_fields = ["stage_id", "team_id"] + self._pls_get_safe_fields()
+        use_tags = bool(pls_fields.pop(pls_fields.index('tag_ids')) if 'tag_ids' in pls_fields else False)
```

Comment:
maybe this one liner will be hated by @awa-odoo . I could understand if he would want to split this into multi lines clearer instructions :D

Reply:
bon allez, replace this by :
```
use_tags = 'tag_ids' in pls_fields
if use_tags:
    pls_fields.remove('tag_ids')
```

Reply:
done